src/main/java/mate/academy/bookstore/service/impl/CartItemServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package mate.academy.bookstore.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.cart.CartItemRequestDto;
+import mate.academy.bookstore.dto.cart.CartItemResponseDto;
+import mate.academy.bookstore.exception.EntityNotFoundException;
+import mate.academy.bookstore.mapper.CartItemMapper;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.model.CartItem;
+import mate.academy.bookstore.model.ShoppingCart;
+import mate.academy.bookstore.repository.book.BookRepository;
+import mate.academy.bookstore.repository.cart.CartItemRepository;
+import mate.academy.bookstore.service.CartItemService;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Service
+public class CartItemServiceImpl implements CartItemService {
```

Comment:
I suggest to unite cart Item service and shopping cart service, this entity is very tight related, so it will be easier and more convenient to make invitations over them in one service